src/bin/hq.rs

```diff
@@ -70,6 +71,9 @@ struct ServerStartOpts {
     /// Hostname/IP of the machine under which is visible to others, default: hostname
     #[clap(long)]
     host: Option<String>,
+
+    #[clap(long)]
+    idle_timeout: Option<ArgDuration>,
```

Comment:
Hmm, I would expect this to be a parameter of the worker, not the server :thinking:

Reply:
I was thinking about it as kind of "session" parameter where session is a single start of server. But maybe we can have both server setting (that provides a default) and worker setting that may override the default